<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaListaSimple</class>
 <widget class="QDialog" name="VentanaListaSimple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="InsertarInicioBTN">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>insertar al inicio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InsertarFinalBTN">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>insertar al final</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InsertarPOSBTN">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>insertar en un posición</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EliminarInicioBTN">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>eliminar al inicio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EliminarFinalBTN">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>eliminar al final</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EliminarPOSBTN">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>eliminar por posición</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EditarPosBTN">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>editar por posición</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ObtenerPosBTN">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>obtener por posición</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar número</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="InsertarNumero">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Número</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Posición Insertar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="PosicionInsertar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>551</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Posicion eliminar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="PosEliminar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="PosEditarObtener">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Posición</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Número</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumeroEditar">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lista Simple</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VentanaListaSimple</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VentanaListaSimple</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
